jnigen/lib/src/summary/summary.dart

```diff
@@ -86,3 +91,85 @@ class SummarizerCommand {
     return proc;
   }
 }
+
+Future<Classes?> getSummary(Config config) async {
```

Comment:
This can return a `Future<Classes>` instead. The code after `log.fatal` never runs and only then you're returning null.

Reply:
I need to still return a Future.error. (Otherwise compiler will not know that `log.fatal` always exits).

Reply:
Wait I can make `log.fatal` return `Never`, right?